Pseudocode/Stack/Stock Span/SourceCode.tex

```diff
@@ -0,0 +1,67 @@
+% Set the Page Layout
+\documentclass[12pt]{article}
+\usepackage[inner = 2.0cm, outer = 2.0cm, top = 2.0cm, bottom = 2.0cm]{geometry}
+
+
+% Package to write pseudo-codes
+\usepackage{algorithm}
+
+% Don not remove the 'end' at the end of the algorithm
+\usepackage{algpseudocode}
+\usepackage{xcolor}
+
+% Let us keep the 'end only for 'For'
+% Manually remove the 'end' for some sections
+\algtext*{EndIf}
+\algtext*{EndFunction}
+
+% Define Left Justified Comments
+\algnewcommand{\LeftComment}[1]{\Statex \(\triangleright\) #1}
+
+% Remove the Numbering of the Algorithm
+\usepackage{caption}
+\DeclareCaptionLabelFormat{algnonumber}{Algorithm}
+\captionsetup[algorithm]{labelformat = algnonumber}
+
+\newcommand{\IsNotEmpty}{is \textbf{not empty, }}
```

Comment:
Remove the space after `,` . It is giving one extra space (as you can see below)